be/src/column/decimalv3_column.cpp

```diff
@@ -82,36 +82,14 @@ void DecimalV3Column<T>::crc32_hash(uint32_t* hash, uint32_t from, uint32_t to)
 }
 
 template <typename T>
-int64_t DecimalV3Column<T>::xor_checksum() const {
+int64_t DecimalV3Column<T>::xor_checksum(uint32_t from, uint32_t to) const {
     // The XOR of DecimalV3Column
     // XOR all the decimals one by one
     auto& data = this->get_data();
     int64_t xor_checksum = 0;
-    size_t num = data.size();
     const T* src = reinterpret_cast<const T*>(data.data());
 
-#ifdef __AVX2__
```

Comment:
why did you remove SIMD code?

Reply:
The compiler has already optimized the easy loop. And I test also it has no performance differenece.